### DISCUSSION TITLE
Bypass/Disable plugin in chain

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1206] [Help]
It would be great to be able to fully bypass (or disable as seen in some DAW) plugins in a chain.

I see at least two reasons for that : 
- currently bypassed plugin in AG chain are still affecting the whole latency
- CPU usage is not reduced when bypassing plugin in an AG chain

**Comment by apohl79** [comment]
When you bypass a plugin it will be released actually and should release its resources stop consuming CPU. It is not unloaded though, but I think that would be too heavy. The latency is kept the same and is compensated by the server. I found that this is less impactful as changing the latency can be auditable.
